packages/button-theme/src/DatePicker.tsx

```diff
@@ -41,6 +41,11 @@ export const datepickerStyles = {
       `,
     },
   },
+  rounded: {
+    input: `
+      border-radius: 0.25em;
+    `,
+  },
```

Comment:
Can we add this to the global style object instead of the `DatePicker` component specifically? Or will that break other components' styles?